docs/db_structure.md

```diff
@@ -64,6 +65,7 @@ CREATE TABLE rule_error_key (
     publish_date  TIMESTAMP NOT NULL,
     active        BOOLEAN NOT NULL,
     generic       VARCHAR NOT NULL,
+    tags VARCHAR NOT NULL DEFAULT '',
```

Comment:
Do we need this? Because before it worked well and no new features that use this `tags` was not added.

Reply:
This did exist, I just updated the docs to be the same as the state of the DB after all applied migrations :)

Reply:
Oh, ok, thank you =)